docs/GitHubActions.md

```diff
@@ -0,0 +1,44 @@
+# GitHub Actions Backend
+
+## Getting Started
+The easiest way to use `sccache` with GitHub Actions is to use the [GitHub Action](https://github.com/marketplace/actions/rust-cache-using-sccache).
+The action will automatically download and install `sccache` and configure it to use the GitHub Actions cache.
+It will add `RUSTC_WRAPPER` environment to your build steps to use `sccache` as the compiler wrapper.
+
+Just add this step to your workflow:
+
+```yaml
+- name: Cache Rust compilation
+  uses: metalbear-co/sscache-action@v1.0.2
```

Comment:
Sorry but I am not confortable recommending this
Could you please remove this?

Reply:
I was about suggest that, and invest a few hours into vetting and polishing so we can endorse something that's easier to use compared to a DIY manual.